versioned_docs/version-latest/develop/nova/quick_start.md

```diff
@@ -33,11 +33,12 @@ You can use **[Remix](https://remix.ethereum.org/)**, **[Foundry](https://book.g
 If anything above sounds unfamiliar, you can always fall back to our full guide. 
 
 ---
-### Important note about submitting a transaction
+## Important note about submitting a transaction
```

Comment:
I think we need to keep the note slightly smaller, aka using ###

Reply:
Feel free to ignore it, not a deal breaker for now.

Reply:
Fixed